src/components/DisplayJobs/index.js

```diff
@@ -0,0 +1,69 @@
+import './displayjobs/jobs.css'
+
+const DisplayJobs = ({setSelectedID, selectedID, displayJobs, setDisplayJobs}) => {
+
+    return (
+        <div className="container px-5">
+            <h3 className="fw-bold py-3">Most recent jobs</h3>
+            <table className="table table-dark table-striped">
+                <thead>
+                <tr>
+                    <th className="col-5" colSpan="2" scope="col">
+                        Job Title/ Company
+                    </th>
+                    <th className="col-2" scope="col">
+                        Type
+                    </th>
+                    <th className="col-2" scope="col">
+                        Salary
+                    </th>
+                    <th className="col-3" scope="col">
+                        Skills
+                    </th>
+                </tr>
+                </thead>
+                <tbody>
+                    {displayJobs.map((job, key) => {
+                        const roundedSalary = job.salary ? Number(job.salary.toPrecision(2)) : ("")
+                        return (
+                            <tr key={key}>
+                            <td scope="row">
```

Comment:
Consistency: The following content should be indented within the `<tr>`.

Reply:
Done